### ISSUE TITLE
Following the recommended steps to change putty settings doesn't affect default window behaviour

### ISSUE THREAD

**Issue by Sturmey66** [mRemoteNG/mRemoteNG#2226]
When opening a session, I always get courier new 10pt. Doesn't matter what I use in default settings in putty, always courier new 10pt.
I can change the window by right clicking the tab and changing the settings or by choosing load default, but every window every time loads courier new 10 pt.

## Expected Behavior
Based on what ever help file and every forum post says, I should be able to save the default settings with a different color, font, background or something else, and have new windows open with those settings.

## Current Behavior
Regardless of what is set in default settings in putty, every window in mRemoteNG opens with black background, courier new font at 10pt size.

## Possible Solution
update help to show where to make the change without opening putty?

## Steps to Reproduce (for bugs)

1.on my system, open mRemoteNG, open SSH to server
2.session opens and it's Courier new font 10pt
3.
4.

## Context
It's a bit hard to read 10pt and I'd like to default to 12pt

## Your Environment
Host Name:                 WORK-PC
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.19044 N/A Build 19044
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          user
Registered Organization:
Product ID:                00342-50687-60031-AAOEM
Original Install Date:     2021-10-20, 16:20:15
System Boot Time:          2022-05-14, 00:39:24
System Manufacturer:       Dell Inc.
System Model:              Precision 7510
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 94 Stepping 3 GenuineIntel ~2701 Mhz
BIOS Version:              Dell Inc. 1.21.3, 2020-08-04
Windows Directory:         C:\Windows
System Directory:          C:\Windows\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC-06:00) Central Time (US & Canada)
Total Physical Memory:     16,275 MB
Available Physical Memory: 8,416 MB
Virtual Memory: Max Size:  22,675 MB
Virtual Memory: Available: 9,035 MB
Virtual Memory: In Use:    13,640 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\WORK-PC
Hotfix(s):                 15 Hotfix(s) Installed.
                           [01]: KB5013624
                           [02]: KB4562830
                           [03]: KB4570334
                           [04]: KB4577266
                           [05]: KB4577586
                           [06]: KB4580325
                           [07]: KB5003791
                           [08]: KB5007115
                           [09]: KB5013942
                           [10]: KB5006753
                           [11]: KB5007273
                           [12]: KB5011352
                           [13]: KB5011651
                           [14]: KB5014032
                           [15]: KB5005699
Network Card(s):           9 NIC(s) Installed.
                           [01]: Intel(R) Ethernet Connection (2) I219-LM
                                 Connection Name: Ethernet 2
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.1.1
                                 IP address(es)
                                 [01]: 192.168.1.105
                                 [02]: fe80::60b7:92fe:52bf:ad28
                           [02]: Intel(R) Dual Band Wireless-AC 8260
                                 Connection Name: Wi-Fi
                                 Status:          Media disconnected
[03]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Bluetooth Network Connection
                                 Status:          Media disconnected
                           [04]: Wintun Userspace Tunnel
                                 Connection Name: OpenVPN Wintun
      Status:          Media disconnected
Hyper-V Requirements:      VM Monitor Mode Extensions: Yes
                           Virtualization Enabled In Firmware: Yes
                           Second Level Address Translation: Yes
                           Data Execution Prevention Available: Yes

**Comment by simonai1254** [comment]
Hi 

How do you try to set the Putty settings? First of all you should save the connection profile in the Putty environment that mRemoteNG uses, which can be accessed via File>Options>Advanced>Launch Putty. 

Important then is to do the changes and save it to a profile you'd like, this may also be the default settings. 
Next verify that in the connection configuration the Protocol is set to "SSH version 2" and the PuTTY settings are set to the profile you saved. 

That's it, nothing special :). From now on your SSH-sessions will take the saved settings. 

Hope this helps and best regards 
Simon

**Comment by Sturmey66** [comment]
I did all that, and no change. everything I did to try and set the default settings to a different font made no difference. 

I did find a solution for anyone else in this situation. I had used putty directly and not through mRemoteNG so there were other saved profiles in putty. I removed all other saved settings and now it appears that the "Default Settings" profile applies. Something in how mRemoteNG loads settings means that if there is more than one profile in putty it might not take the one labeled "Default Settings" for it's options.

I'm rather disappointed that nobody else had found that in the past. Hopefully this post will get discovered by search engines for the next person with this issue.
